internal/backends/postgresql/collection.go

```diff
@@ -312,42 +312,45 @@ func (c *collection) Explain(ctx context.Context, params *backends.ExplainParams
 
 // Stats implements backends.Collection interface.
 func (c *collection) Stats(ctx context.Context, params *backends.CollectionStatsParams) (*backends.CollectionStatsResult, error) {
-	p, err := c.r.DatabaseGetExisting(ctx, c.dbName)
-	if err != nil {
-		return nil, lazyerrors.Error(err)
-	}
+	if params.Refresh {
```

Comment:
If `params.Refresh` is true, we should call [`ANALYZE`](https://www.postgresql.org/docs/current/sql-analyze.html) on that collection/table, then calculate stats as before

Reply:
So the stats func should continue its execution and remain unchanged,  should i be modifying the `collectionStat` function and add a parameter, for the params.refresh to add checks locally where the `ANALYZE` is called? 
I will add a commit for these changes

Reply:
```go
// get p, coll as before

if params.Refresh {
  // call ANALYZE
}

stats, err := collectionsStats(...
```